deployment/SampleClaimsProvider/.gitignore

```diff
@@ -0,0 +1,7 @@
+*.user
```

Comment:
Do we need another .gitignore or are these covered in a higher level .gitignore?